spec/forms/create_video_form_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.describe "youtubeのurlをcodeに変換" do
```

Comment:
クラス名にするのが慣例

```suggestion
RSpec.describe CreateVideoForm do
```

クラス名